_data/server_permissions.yaml

```diff
@@ -0,0 +1,151 @@
+# general permissions
+- permission: adminMessageReceive
+  description: Receives messages sent to the admin channel
+- permission: adminMessageSend
+  description: Allows sending messages to the admin channel
+- permission: hideAdmin
+  description: Doesn't show `@`
+- permission: info
+  description:
+- permission: privateMessage
+  description: Allows sending private messages with `.`
+- permission: rejoin
+  description: Allows rejoining instantly, regardless of `_rejoinTime`
+- permission: removePerms
+  description: Allows removing permissions from players
+- permission: requireIdentify
+  description: Requires callsign to identify before allowed to spawn
+- permission: setPerms
+  description: Alls granting permissions to players
+- permission: shortBan
+  description: Allows bans with limited duration only
+- permission: showAdmin
+  description: Gives player an `@` regardless of having the `shortban` or `ban` permission; the `hideAdmin` permission supersedes the `showAdmin` permission
+- permission: spawn
+  description: Allows joining the game as a player; observers don't need this permission
+- permission: talk
+  description: Allows sending messages
+
+# slash commands
```

Comment:
this is more for @allejo but should the linking of permissions to commands be done here, or in the commands files? I'd think the latter is more maintainable, but I'm not sure

Reply:
Oooo you're entirely right! I completely forgot we had already semi-planned for this; e.g. [`_documentation/user/slash_commands/ban.md`](https://github.com/BZFlag-Dev/bzflag.org/blob/master/_documentation/user/slash_commands/ban.md).

We could make use of `where` for this. For example,

```twig
collections.slash_commands | where('permissions', '_=', 'ban')
```

would give us all of the commands that require the "ban" permission.

Reply:
Done in ee8813d97f96c0316e6a4a22d081ff542d24d293. I like this idea much better, I even see some slash commands that were missed at the beginning of this PR. Good call @jwmelto! :smile: